### DISCUSSION TITLE
Why does the following error occur? `* '[base].exclude' expected type 'string', got unconvertible type '[]interface {}', value: '[examples/** docs/**]'`

### DISCUSSION THREAD

**Discussion by toshiki31** [yamada-ui/yamada-ui#4188] [Q&A] [answered]
When I created Pull Request #4187 and ran `git push`, the following error occurred:

```
Error: 1 error(s) decoding:

* '[base].exclude' expected type 'string', got unconvertible type '[]interface {}', value: '[examples/** docs/**]'

```

Upon investigation, I found that this error is likely caused by specifying `exclude` as an array in `yamada-ui/lefthook.yaml`. In fact, although it’s not syntactically ideal, changing `exclude` from an array to a string made `git push` succeed.

However, the syntax in `yamada-ui/lefthook.yaml` itself seems correct. So why does this error occur? Is there something in my development environment that prevents the file structure from being recognized correctly?

**Comment by hirotomoyamada** [comment]
@toshiki31 

Hi, thanks for the comment!

The `v2.0` branch is currently under development, and most of the type checking and scripting is not working properly.

**Reply by toshiki31** [reply]
Thank you for replying my comment.
I understand.

**Reply by toshiki31** [reply, marked as answer]
This issue has been resolved. The version of lefthook I was using was 1.6.0, which did not support the exclude option introduced in version 1.7.3 and later. This was the cause of the problem.

https://github.com/evilmartians/lefthook/releases/tag/v1.7.3
